woocommerce/woocommerce-admin
# Refactor package style builds

This PR seeks to clean up the SASS compilation for both packages and the main app. The main goal is to remove the `bin/packages/*` scripts, but additionally it:

* Creates a private `@woocommerce/style-build` package that exports a common Webpack config for building stylesheets
* Moves the common dependencies from `client/stylesheets/abstracts` to the new package
* Cleans up defunct IE style enqueue calls
* Adds a missing RTL definition for the `onboarding` styles
* Adds `tsconfig.tsbuildinfo` to the `clean` scripts (unrelated to styles of course, but was found in the course of testing the `start` scripts)

### Detailed test instructions:

- `npm run clean` should delete `tsconfig.tsbuildinfo` files
- `npm run build`
- Smoke test app and make sure styles are correct
- `npm start`
- Make both a JS and SCSS change in a package and in the `client/`
- Verify the changes are reflected after a browser refresh
